### ISSUE TITLE
[4.12 installer] OKD on Openstack - dashboard for Cluster Settings -> Details doesn't work

### ISSUE THREAD

**Issue by alexiusflavius** [okd-project/okd#1478]
**Describe the bug**

Following the successful installation of OKD 4.12 in Openstack I found that dashboard page for Cluster Settings -> Details has the following issue:
`"Oh no! Something went wrong."`

**Version**
4.12.0-0.okd-2023-01-21-055900

**How reproducible**
Follow the basic deployment pattern using install-config.yaml

**Log bundle**

TypeError
Description:

r is null
Component trace:
```
ke@https://console-openshift-console.apps.ocp-4-12.idm.local/static/cluster-operator-chunk-2af5dd6e84c407a5eade.min.js:1:23212
div
div
div
div
div
Ve@https://console-openshift-console.apps.ocp-4-12.idm.local/static/cluster-operator-chunk-2af5dd6e84c407a5eade.min.js:1:28928
i@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:469890
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:58042
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:60087
Suspense
div
R@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:185359
N@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:185854
div
A<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:855114
S<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:497526
S@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:93:80887
@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:499803
S@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:93:80887
Re@https://console-openshift-console.apps.ocp-4-12.idm.local/static/cluster-operator-chunk-2af5dd6e84c407a5eade.min.js:1:35106
s@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:236886
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:58042
T
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:60087
Suspense
i@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:469890
section
m@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:460022
div
div
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1499355
div
div
c@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:318197
d@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:1:14313
div
d@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:285416
l@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1156919
H<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:476744
S@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:93:80887
main
div
v@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:47205
div
div
c@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:1:155068
div
div
c@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:318197
d@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:1:14313
div
d@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendor-patternfly-core-chunk-5d366f9ff45bc8b789ae.min.js:5:285416
On@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:36:185080
t.default@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:859109
5244/t.default@https://console-openshift-console.apps.ocp-4-12.idm.local/static/quick-start-chunk-b1d3ca0d8c571e2cc732.min.js:1:1264
s@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:236886
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1570211
Z@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1587902
st@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:36:145279
Z@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1587902
Z@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1587902
i@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:816211
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1564576
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1561993
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1564387
ee@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1588044
te<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1590824
r@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:36:125397
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:58042
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:60087
t@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:21:54647
ne@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1591135
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:790846
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:1062123
s@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:612723
t.a@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:141:244866
Suspense
```

Stack trace:
```
E@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:252571
ke@https://console-openshift-console.apps.ocp-4-12.idm.local/static/cluster-operator-chunk-2af5dd6e84c407a5eade.min.js:1:23683
na@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:58879
Hs@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:111315
wc@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:98329
Cc@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:98257
_c@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:98118
pc@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:95105
Vi/<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:44774
t.unstable_runWithPriority@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:262:3768
Hi@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:44551
Vi@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:44721
qi@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:44654
sc@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:92482
enqueueSetState@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:254:48628
m.prototype.setState@https://console-openshift-console.apps.ocp-4-12.idm.local/static/vendors~main-chunk-c0c15fae48996264a3e4.min.js:246:1306
loadComponent/<@https://console-openshift-console.apps.ocp-4-12.idm.local/static/main-chunk-f1b3a746329e0eefe97d.min.js:1:237542
```

**Comment by vrutkovs** [comment]
@jhadvig could you look into this? 

I guess we'll need must-gather to see cluster details too

**Comment by vrutkovs** [comment]
Does it work if you unset overrides?
```
    - group: config.openshift.io
      kind: Ingress
      name: cluster
      namespace: ""
      unmanaged: true
```
is probably incorrect - this resource is not namespaced, so perhaps `namespace: ""` triggers it?

**Comment by alexiusflavius** [comment]
Hello - jsut has an ability to check this - indeed removing overrides helped restoring the page
Thou I wonder why those overrides were left there in the first place
Thanks

**Comment by vrutkovs** [comment]
https://issues.redhat.com/browse/OCPBUGS-6678 is upstream issue - a combination of "no available updates" and "has overrides" triggered it. The fix landed in 4.12 nightlies
